packages/dht/src/dht/DhtNode.ts

```diff
@@ -75,7 +75,7 @@ export interface DhtNodeOptions {
     websocketHost?: string
     websocketPortRange?: PortRange
     websocketServerEnableTls?: boolean
-    peerId?: DhtAddress
+    id?: DhtAddress
```

Comment:
I would name this as `nodeId`. I think the `DhtNode` class is so large and has so many responsibilities that it would not be apparent that `id` would refer to `nodeId`.

Reply:
Changed